routes/user.js

```diff
@@ -1,16 +1,33 @@
-const router = require('express').Router();
+const express = require('express');
+const router = express.Router();
 const userCtrl = require('../controllers/user');
+const create = require('../models/workoutlog');
+
+
+// router.get("/", function (req, res) {
+//   console.log(req.user, "index")
+//   res.render("user", {
+//     user: req.user,
+//   });
+// });
+
 
 
 // GET /user
-router.get('/user', isLoggedIn, userCtrl.index);
+router.get('/', isLoggedIn, userCtrl.index);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
router.get("/", isLoggedIn, userCtrl.index);
```